src/CodeGen_Hexagon.cpp

```diff
@@ -531,6 +534,8 @@ void CodeGen_Hexagon::init_module() {
 
         // Broadcasts
         { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatw), u32v1,  "splat.w", {u32} },
+        { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatb), u8v1,   "splat.b", {u8}  },   // v62 - Broadcasts
```

Comment:
@dsharletg: I don't know if module->getFunction respects weak linkage in that way. Does it? See -> http://llvm.org/docs/doxygen/html/Module_8cpp_source.html#l00191  getFunction seems to use getNamedValue that returns the first global value in the module with the specified name. (http://llvm.org/docs/doxygen/html/classllvm_1_1Module.html#a75391fa576679482ed9195d69101d002)